src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/Properties/AssemblyInfo.cs

```diff
@@ -20,8 +20,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Batch Management Library")]
 [assembly: AssemblyDescription("Provides management functions for Microsoft Azure Batch services.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
normally, you want AssemblyVersion to just have the the major version and the fileversion to match the package version